addons/web/static/tests/core/expression_editor_dialog.test.js

```diff
@@ -0,0 +1,83 @@
+import { expect, test, describe } from "@odoo/hoot";
+
+import { ExpressionEditorDialog } from "@web/core/expression_editor_dialog/expression_editor_dialog";
+import { contains, defineModels, mountWithCleanup } from "@web/../tests/web_test_helpers";
+import {
+    Partner,
+    Product,
+    getTreeEditorContent,
+} from "@web/../tests/_framework/condition_tree_editor_test_helpers";
+import { animationFrame } from "@odoo/hoot-mock";
+import { makeDialogMockEnv, mockService } from "@web/../tests/_framework/env_test_helpers";
+
+import { Component, xml } from "@odoo/owl";
+
+describe.current.tags("desktop");
+
+async function makeExpressionEditorDialog(params = {}) {
+    const props = { ...params };
+
+    class Parent extends Component {
+        static components = { ExpressionEditorDialog };
+        static template = xml`<ExpressionEditorDialog t-props="expressionEditorProps"/>`;
+        static props = ["*"];
+        setup() {
+            this.expressionEditorProps = {
+                expression: "1",
+                close: () => {},
+                onConfirm: () => {},
+                ...props,
+                resModel: "partner",
```

Comment:
what if we have a test that check on an other model?
move resModel before `...props`